public/src/components/channelManagement/epicTests/__tests__/epicTestVariantEditor.spec.tsx

```diff
@@ -1,19 +0,0 @@
-import { getInvalidTemplateError } from '../../helpers/copyTemplates';
-
-describe('maxViewsEditor tests', () => {
-  describe('getInvalidTemplateError tests', () => {
-    it('should return null if no templates', () => {
-      expect(getInvalidTemplateError('Blah blah')).toEqual(null);
-    });
-
-    it('should return null if template is ok', () => {
-      expect(getInvalidTemplateError('I am from %%COUNTRY_NAME%%')).toEqual(null);
-    });
-
-    it('should return an error message if template is not ok', () => {
-      expect(getInvalidTemplateError('I am from %%COUNTRY_NAM%%')).toEqual(
-        'Invalid template: %%COUNTRY_NAM%%',
-      );
-    });
-  });
-});
```

Comment:
We could move these into a new file e.g `validation.spec.tsx` and test the function in `validation.tsx`?

Reply:
yes - I think test cases are still useful here

Reply:
Yeah makes sense - I'll add them back. I can add a few new cases as `if template is not ok` could mean a few things

Reply:
I've pushed that up!